apps/ledger-live-desktop/src/renderer/index.html

```diff
@@ -173,12 +157,12 @@
         c24.8,0,47.5-9.9,59.1-31.4h5.8v27.3h38V330.1h-112v38H1203.6z M839.2,228.1h45c42.6,0,65.7,10.7,65.7,68.6v76
         c0,57.8-23.1,68.6-65.7,68.6h-45V228.1z M887.9,479.3c78.9,0,108.3-59.9,108.3-144.6c0-85.9-31.4-144.6-109.1-144.6h-89.2v289.2
         H887.9z M598.3,352.9h127.3v-38H598.3v-86.8h139.7v-38H556.1v289.2h188v-38H598.3V352.9z M331.8,190.1h-42.1v289.2h190.1v-38H331.8
-        V190.1z M0,479.3v190.1h289.2v-42.1H42.1V479.3H0z M0,0v190.1h42.1V42.1h247.1V0H0z"
-            />
-          </svg>
-        </div>
+        V190.1z M0,479.3v190.1h289.2v-42.1H42.1V479.3H0z M0,0v190.1h42.1V42.1h247.1V0H0z" />
+        </svg>
       </div>
     </div>
-    <div id="modals"></div>
-  </body>
+  </div>
+  <div id="modals"></div>
+</body>
+
```

Comment:
@elbywan would it be possible not to regen the prettier in context of this PR? 🙏

Reply:
woops yeah I'll fix that 👍